### PR TITLE
DeepCCI removal and out channel parameters

### PR DESCRIPTION
**Pull request by benedekrozemberczki**
# Summary
 
- Removes the DeepCCI class.
- Adds Out channels for DeepSynergy and EPGCN DS.
- Moves EPGCN-DS to GraphConv layers.

- [ ] Code passes all tests
- [x] Unit tests provided for these changes
- [x] Documentation and docstrings added for these changes

### CODE REVIEW

**Comment by cthoyt** [comment]
@benedekrozemberczki flake8 got you!

**Comment by benedekrozemberczki** [comment]
I know.

File: chemicalx/models/epgcnds.py [hunk 1]
```diff
@@ -21,18 +21,21 @@ class EPGCNDS(Model):
        <https://ojs.aaai.org/index.php/AAAI/article/view/7236>`_
     """
 
-    def __init__(self, *, in_channels: int, hidden_channels: int = 32, out_channels: int = 16):
+    def __init__(
+        self, *, in_channels: int, hidden_channels: int = 32, middle_channels: int = 16, out_channels: int = 1
```
Comment (cthoyt) [inline chemicalx/models/epgcnds.py:25]: Consider adding a default value for in_channels?

Comment (benedekrozemberczki) [reply chemicalx/models/epgcnds.py:25]: Yes, why not!

**Review by cthoyt — APPROVED** [review]

**Pull request merged**